File: exo3/exo3.py
#count letter
#by fflachet
#beweb
import sys

def countLetter(word,letter):
	i = 0
	result = 0
	word = sys.argv[1]
	letter = sys.argv[2]
	while i < len(word):
		if word[i] == letter:
			result += 1
		i += 1
	print('le nombre de ' + str(letter)+' est : ' + str(result))

countLetter(sys.argv[1],sys.argv[2])

File: exo3/exo3.rb
#count letter
#by fflachet
#beweb



def countLetter(word,letter)

	result = 0
	i = 0
	word = ARGV[0]
	letter = ARGV[1]

	while i < word.length
		if word[i] == letter
			result+=1
		end
	i+=1
	end
	print "le nombre de #{letter} est : #{result}\n"
end

countLetter(ARGV[0], ARGV[1])